CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### New features
 
 * Change `qos::wal` operator to only require one of `max_elements` or `max_bytes` (using both is still possible).
+* Add `$udp.host` and `$udp.port` to allow controling udp packet destinations on a per event basis.
```

Comment:
while i like the terseness of those names, i would think about making it consistent with the config, where it is `dst_host` and `dst_port`. `host` and `port` are the local socket address, this might be confusing.

Reply:
Ja that's a bit of a conflict, on the other hand in the `rest` metadata we call it `host` and `port` so naming it `dst_host` and `dst_port` will confuse people who used this mechanism from rest before :( 

how about we depreciate `dst_host` and `dst_port` (warning for this release) and add a `dst` struct that would be `dst.host`, `dst.port` that way we can keep short names and stay consistent with the config?

```yaml
offramp:
  - id: udp
    type: udp
    config:
      host: 0.0.0.0
      port: 0
      dst:
        host: snot
        port: 42
```

or we ad `bind` for current `host` and `port` to allow specifying the binding locally?


```yaml
offramp:
  - id: udp
    type: udp
    config:
      bind:
        host: 0.0.0.0
        port: 0
      host: snot
      port: 42
```


(I kind of like the second option since the top-level host/port for the binding address always struck me as wrong)

Reply:
Lets go with the second option. :)
But then we should rename the `bound` config to `connected`, as the setting, if `false`, avoids the `connect` call, not the bind call.

Reply:
Oh that's a good point! If we think further, how about we depricate `bound` altogether with this and enforce re-connects to be done via metadata, that will reinforce the feeling of "this happens on every event since I've to set it on every event"?

Reply:
kill it with fire! :fire: